src/module/item/SR5ItemSheet.ts

```diff
@@ -352,6 +372,24 @@ export class SR5ItemSheet extends ItemSheet {
         this._activateTagifyListeners(html);
     }
 
+    /**
+     * User requested removal of the linked actor.
+     */
+    async handleLinkedActorRemove(event: any) {
+        await this.item.update({ 'system.linkedActor': '' });
+        this.item.render(true);
```

Comment:
render(true) is only necessary to _force_ a render, even if the sheet is not open. In this case it _should_ be open but so longs as there is no need for it, why force it. If you had a use case for this, that I'm not seeing, you can keep it.

Same goes for `updateLinkedActor`

https://foundryvtt.com/api/classes/foundry.applications.api.ApplicationV2.html#render